Resources/Textures/Objects/Weapons/Bombs/c4.rsi/meta.json

```diff
@@ -1,31 +1,35 @@
 {
-    "version": 1,
-	"license": "CC-BY-SA-3.0",
-	"copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/b13d244d761a07e200a9a41730bd446e776020d5",
-    "size": {
-        "x": 32,
-        "y": 32
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/b13d244d761a07e200a9a41730bd446e776020d5",
```

Comment:
Is the helmet also from TGStation? 

You credit `LeveFUN` in the changelog, so I'm assuming they're the artist or something?

Reply:
Yea, he is artist, you right. Forgot about meta licensing